src/i18n.d.ts

```diff
@@ -3,12 +3,37 @@
 
 import { ComponentType, ReactNode } from 'react'
 
+interface InterpolationOptions {
```

Comment:
Is there any reason not to rather `import { InterpolationOptions } from 'node-polyglot'`?

Reply:
Node-polyglot doesn't have official types, its just using the @types/node-polyglot package from DefinitelyTyped. Do you think we should rely on them? They do not have typing for all options such as pluralRules but most of the other ones seem to be correct. I think it would be better to add types to the official node-polyglot package then maybe we could depend on them.